migrator/migrations/n_10_to_n_11_postgres_cluster_health_statuses/postgres/postgres_plugin.go

```diff
@@ -139,6 +139,10 @@ func (s *storeImpl) copyFromClusterHealthStatuses(ctx context.Context, tx *postg
 			return marshalErr
 		}
 
+		if pgutils.NilOrUUID(obj.GetId()) == nil {
```

Comment:
Do we know how does this happen? Why the id is not as expected?

Reply:
@c-du Turns out it is because `n_94` used the wrong bucket name

`clusterHealthStatusBucket = []byte("cluster_health_status")`
instead of
`bucket = []byte("clusters_health_status")`